### PR TITLE
add forum TOS

### PR DESCRIPTION
**Pull request by jospoortvliet**
I thought we had a standard ToS on the forums but that is not the case. So ... I went ahead and put something together that at least covers the basics.

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

File: page-privacy.php [hunk 1]
```diff
@@ -121,6 +121,20 @@
...
+                <li><?php echo $l->t('You must at least be 10 years old');?></li>
+                <li><?php echo $l->t('You may no longer use the forum if Nextcloud GmbH or any of the admins contact you directly to say that you may not.');?></li>
+                <li><?php echo $l->t('Keep in mind our forum is to discuss Nextcloud and related, technical issues. Political debates, complaints or such about things not directly related, or purely relevant for yourself, do not belong on the forums. Respect the decisions of administrators - when asked to not discuss things, do not discuss them on the forums.');?></li>
+                <li><?php echo $l->t('To contribute to our forums, you need an account. For that we need a real email address. You can ask us to close your accounta nd delete all private data (but not the posts, see below) any time. Use the contact form on the bottom of this page. Note that YOU are responsible for all actions taken with your account, whether authorized or not, unless you ask us to close your account or notify us it is compromised. You agree to pick a secure password and keep it safe!');?></li>
```
Comment (stefan-niedermann) [inline page-privacy.php:131]: typo: account**a** nd

**Comment by jospoortvliet** [comment]
> Maybe a paragraph about that a forum is based on content of community - if someone hotlinks an image, then the browser of the visitor will fetch the image and transfer the IP adresse (and some other data) to this server. Give hint that the user can configure his browser or use plugins to not automatically fetch 3rd party content.

Very good, yeah, will do.

File: page-privacy.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
+            <p><?php echo $l->t('<strong>Our forums</strong> on <a class="hyperlink" href="https://help.nextcloud.com">help.nextcloud.com</a> runs the open source <a class="hyperlink" href="https://discourse.org">discourse</a> software. This is self-hosted and we do not share, disclose or otherwise distribute any data from it. You can disable your own account there if you need to. Note that this platform is meant for home users and home user questions! As enterprise visitor, please use <a class="hyperlink" href="https://portal.nextcloud.com">our support portal</a>. If you still use the forums, expect that we reach out to you at some point.');?> <?php echo $l->t(' To use our forums you must agree with our Terms of Service. See below for the complete TOS for the use of our forums.');?><a class="hyperlink" href="#TOS">Terms of Service</a></p>
```
Comment (tcitworld) [inline page-privacy.php:27]: `Terms of Service` needs a translation

File: page-privacy.php [hunk 3]
```diff
@@ -120,7 +120,21 @@
...
+                <li><?php echo $l->t('To contribute to our forums, you need an account. For that we need a real email address. You can ask us to close your account and delete all private data (but not the posts, see below) any time. Use the contact form on the bottom of this page. Note that YOU are responsible for all actions taken with your account, whether authorized or not, unless you ask us to close your account or notify us it is compromised. You agree to pick a secure password and keep it safe!');?></li>
+                <li><?php echo $l->t('You agree all content posted on the forums is owned by you and by posting on our forums is shared and licensed under the Attribution-ShareAlike 4.0 International license.');?> <a class="hyperlink" href="http://creativecommons.org/licenses/by-sa/4.0/">Attribution-ShareAlike 4.0 International</a></li>
+                <li><?php echo $l->t('You may not break the law using the forum, use other users accounts, trade, buy or sell user names, advertise any products or use the forums to gather information for advertisement purposes. Also, you man not imply you are affiliated with or endorsed by Nextcloud GmbH or impersonate others.');?></li>
```
Comment (tcitworld) [inline page-privacy.php:133]: `man` :arrow_right: `must`

File: page-privacy.php [hunk 4]
```diff
@@ -120,7 +120,21 @@
...
+                <li><?php echo $l->t('You may not automate access to the forum or monitor it with tools like browser plug-ins, web crawlers and the like, unless it is for the purposes of a search engine. You may not circumvent security or access restrictions or try to work around our moderation efforts. Do not strain the forum with tecnical requests or requests designed to impose an unreasonable load on the forum infrastructure.');?></li>
```
Comment (tcitworld) [inline page-privacy.php:134]: `tecnical` :arrow_right: `technical`

**Pull request merged**